docs/integrations/github/org.md

```diff
@@ -91,6 +93,7 @@ Directly under the `githubOrg` is a list of configurations, each entry is a stru
 - `id`: A stable id for this provider. Entities from this provider will be associated with this ID, so you should take care not to change it over time since that may lead to orphaned entities and/or conflicts.
 - `githubUrl`: The target that this provider should consume
 - `orgs` (optional): The list of the GitHub orgs to consume. If you only list a single org the generated group entities will use the `default` namespace, otherwise they will use the org name as the namespace. By default the provider will consume all accessible orgs on the given GitHub instance (support for GitHub App integration only).
+- `alwaysUseDefaultNamespace` (optional): If set to `true`, the provider will always use the `default` namespace for the generated group entities. Groups with the same name across different orgs will be considered the same group. Defaults to `false`.
```

Comment:
@axdotl `alwaysUseDefaultNamespace` is considered as an internal option of `GithubMultiOrgEntityProvider` but is not configurable at the moment, so the changes in this PR aren't correct

Reply:
Oh okay - that was not clear for me when reading the changelog. Also `yarn backstage-cli config:check` didn't show an issue, that's why I assumed it is a valid configuration.

So should I just close the PR instead?

Reply:
yeah I think `config:check` didn't show any issue because your config is still valid if you add extra stuff.
I will close the PR, but thank you anyway for double checking whether something was missing in the docs 🙏